packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -2635,6 +2635,40 @@ describe('<Autocomplete />', () => {
     });
   });
 
+  describe('prop: onScrollToBottom', () => {
+    it('should call onScrollToBottom when scroll reaches bottom', function test() {
+      if (/jsdom/.test(window.navigator.userAgent)) {
+        this.skip();
+      }
+      const onScrollToBottom = spy();
+      const { getByRole } = render(
+        <Autocomplete
+          open
+          options={['one', 'two', 'three', 'four', 'five']}
+          ListboxProps={{ style: { height: '100px' } }}
+          renderInput={(params) => <TextField {...params} />}
+          onScrollToBottom={onScrollToBottom}
+        />,
+      );
+      const textbox = getByRole('combobox');
+      act(() => {
+        textbox.focus();
+      });
+      const listbox = getByRole('listbox');
+      const option = screen.getAllByRole('option')[0];
+      const { height: optionHeight } = option.getBoundingClientRect();
+
+      const listboxPaddingBottom = Number(
+        window.getComputedStyle(listbox).getPropertyPriority('padding-bottom').replace('px', ''),
```

Comment:
I think I got the issue. This should be `getPropertyValue`, not `getPropertyPriority`. `getPropertyPriority` returns the priorities explicitly set on the CSS property (eg: `important`).

 ```suggestion
        window.getComputedStyle(listbox).getPropertyValue('padding-bottom').replace('px', ''),
```

Does this work? It should fail when the options are only 1 `(optionHeight + listboxPaddingBottom)` or when `(optionHeight * 2)` without adding listboxPaddingBottom.

Reply:
yes it worked, thanks for taking a look, i'll add few more test cases and make PR ready for review. If you any more suggestion feel free to comment.